python/py-nbxmpp/Portfile

```diff
@@ -26,9 +26,9 @@ master_sites        pypi:n/nbxmpp/
 
 distname            nbxmpp-${version}
```

Comment:
`master_sites` and `distname` are set automatically by the `python` PG; since these are the defaults you can remove both lines